experiment/nursery/create_bazel_cache_rcs.sh

```diff
@@ -29,6 +33,11 @@ make_bazel_rc () {
     echo "build --remote_http_cache=${CACHE_HOST}" >> $1
     # don't fail if the cache is unavailable
     echo "build --remote_local_fallback" >> $1
+    # make sure the cache considers host toolchain versions
+    GCC_VERSION=$(package_version gcc)
+    echo "build --action_env=__CACHE_GCC_VERSION__=${GCC_VERSION}" >> $1
+    PYTHON_VERSION=$(package_version python)
+    echo "build --action_env=__CACHE_PYTHON_VERSION__=${PYTHON_VERSION}" >> $1
```

Comment:
I think you also need `build --experimental_strict_action_env` for these to take effect.. https://docs.bazel.build/versions/master/command-line-reference.html

Reply:
nope..